CrashReporter.xcodeproj/project.pbxproj

```diff
@@ -3714,7 +3714,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "FRAMEWORK=\"${BUILT_PRODUCTS_DIR}/${PRODUCT_NAME}.framework\"\n\nlipo \\\n\"${BUILD_DIR}/${CONFIGURATION}-iphoneos/libCrashReporter-iPhoneOS.a\" \\\n\"${BUILD_DIR}/${CONFIGURATION}-iphonesimulator/libCrashReporter-iPhoneSimulator.a\" \\\n-create -output \"${FRAMEWORK}/Versions/Current/${PRODUCT_NAME}\"\n\n# Strip debugging symbols\n#strip -S \"${FRAMEWORK}/Versions/Current/${PRODUCT_NAME}\"\n\n# Create a Bitcode backward compatible binary with older Xcode versions but still support architectures from newer Xcode versions\nif [ -z \"$PL_ARM64E_XCODE_PATH\" ]; then\necho \"Path to Xcode version supporting arm64e not provided; Bitcode compatibility and device architectures are limited to the running Xcode version.\"\nelse\nLIB_IPHONEOS_TEMP=\"${BUILD_DIR}/${CONFIGURATION}-iphoneos/libCrashReporter-iPhoneOS.a\"\nLIB_IPHONEOS_FINAL=\"${FRAMEWORK}/Versions/Current/${PRODUCT_NAME}\"\n\n# Build with the Xcode version supporting arm64e\nenv DEVELOPER_DIR=\"$PL_ARM64E_XCODE_PATH\" /usr/bin/xcodebuild ARCHS=\"arm64e\" -configuration Release -target 'CrashReporter-iOS-Device'\n\n# Concatenate the architectures built from Xcode versions \nenv DEVELOPER_DIR=\"$PL_ARM64E_XCODE_PATH\" /usr/bin/lipo ${LIB_IPHONEOS_FINAL} ${LIB_IPHONEOS_TEMP} -create -output ${LIB_IPHONEOS_FINAL}\nfi\n\ncd \"${FRAMEWORK}\" && ln -sf \"Versions/Current/${PRODUCT_NAME}\" ./\n";
+			shellScript = "cd \"${BUILT_PRODUCTS_DIR}\"\n\nlipo \\\n  \"${BUILD_DIR}/${CONFIGURATION}-iphoneos/libCrashReporter-iphoneos.a\" \\\n  \"${BUILD_DIR}/${CONFIGURATION}-iphonesimulator/libCrashReporter-iphonesimulator.a\" \\\n  -create -output ${EXECUTABLE_PATH}\n\n# Create a Bitcode backward compatible binary with older Xcode versions but still support architectures from newer Xcode versions\nif [ -z \"$PL_ARM64E_XCODE_PATH\" ]; then\necho \"Path to Xcode version supporting arm64e not provided; Bitcode compatibility and device architectures are limited to the running Xcode version.\"\nelse\n\n# Build with the Xcode version supporting arm64e\nenv DEVELOPER_DIR=\"$PL_ARM64E_XCODE_PATH\" /usr/bin/xcodebuild ARCHS=\"arm64e\" OBJROOT=\"${OBJROOT}/DependentBuilds\" -configuration Release -target 'CrashReporter-iOS-Device'\n\n# Concatenate the architectures built from Xcode versions \nenv DEVELOPER_DIR=\"$PL_ARM64E_XCODE_PATH\" /usr/bin/lipo \\\n  ${EXECUTABLE_PATH} \\\n  \"${BUILD_DIR}/${CONFIGURATION}-iphoneos/libCrashReporter-iphoneos.a\" \\\n  -create -output ${EXECUTABLE_PATH}\n\nfi\n\nln -sfh Versions/${CURRENT_VERSION}/${PRODUCT_NAME} ${FULL_PRODUCT_NAME}/${PRODUCT_NAME}\n";
```

Comment:
Highlighting changes.
Changed
`shellScript = "FRAMEWORK="${BUILT_PRODUCTS_DIR}/${PRODUCT_NAME}.framework"` to `"cd "${BUILT_PRODUCTS_DIR}"`

`libCrashReporter-iPhoneOS.a`>`libCrashReporter-iphoneos.a`
`libCrashReporter-iPhoneSimulator.a`>`libCrashReporter-iphonesimulator.a`

changed lipo `-output` from `"${FRAMEWORK}/Versions/Current/${PRODUCT_NAME}"` to `${EXECUTABLE_PATH}`